common/app/conf/switches/ServerSideExperimentSwitches.scala

```diff
@@ -12,4 +12,14 @@ trait ServerSideExperimentSwitches {
     sellByDate = never,
     exposeClientSide = false,
   )
+
+  val InteractiveLibrarianAdminRoutes = Switch(
+    SwitchGroup.ServerSideExperiments,
```

Comment:
Maybe we should make this a FeatureSwitch at some point instead of an experiment?

Reply:
Oh yeah... I thought about it..., could not make up my mind in one way or another. Let me update.